assignment-section-05/classification_model/train_pipeline.py

```diff
@@ -0,0 +1,32 @@
+from sklearn.model_selection import train_test_split
+
+from config.core import config
+from pipeline import price_pipe
+from processing.data_manager import load_dataset, save_pipeline
+
+
+def run_training() -> None:
+    """Train the model."""
+
+    # read training data
+    data = load_dataset(file_name=config.app_config.training_data_file)
+
+    # divide train and test
+    X_train, X_test, y_train, y_test = train_test_split(
+        data[config.model_config.features],  # predictors
+        data[config.model_config.target],
+        test_size=config.model_config.test_size,
+        # we are setting the random seed here
+        # for reproducibility
+        random_state=config.model_config.random_state,
```

Comment:
supposedly, because we set the random_state, it should always return the same test/train split.